src/main/java/exceptions/DeadlineException.java

```diff
@@ -0,0 +1,19 @@
+package exceptions;
+
+import util.Utility;
+
+/**
+ * Class for a deadline exception.
+ */
+public class DeadlineException extends MizzException {
```

Comment:
I like how you created Exception subclasses for each of the possible exception cases